kompose.spec

```diff
@@ -52,26 +52,21 @@
 %global testflags -race -cover -v
 
 Name:           kompose
-Version:        1.0.0
-Release:        2%{?dist}
+Version:        1.1.0
+Release:        0.1%{?dist}
 Summary:        Tool to move from 'docker-compose' to Kubernetes
 License:        ASL 2.0
 URL:            https://%{provider_prefix}
 Source0:        https://%{provider_prefix}/archive/%{commit}/%{repo}-%{shortcommit}.tar.gz
 
 # e.g. el6 has ppc64 arch without gcc-go, so EA tag is required
-ExclusiveArch:  %{ix86} x86_64 %{arm} aarch64 s390x
+ExclusiveArch:  %{?go_arches:%{go_arches}}%{!?go_arches:%{ix86} x86_64 aarch64 %{arm}}
```

Comment:
@praveenkumar should I keep it as it was before ?

Reply:
This is auto generated by gofed

Reply:
@surajssd yes please keep it as it was before so fedora builds works as expected.

Reply:
done